.ci/install_nemu.sh

```diff
@@ -19,21 +19,23 @@ arch=$("${cidir}"/kata-arch.sh -d)
 install_nemu() {
 	local nemu_repo=$(get_version "assets.hypervisor.nemu.url")
 	local nemu_version=$(get_version "assets.hypervisor.nemu.version")
+	PACKAGING_REPO="github.com/kata-containers/packaging"
+	NEMU_TAR="kata-nemu-static.tar.gz"
 	case "$arch" in
 	x86_64)
-		local nemu_bin="qemu-system-${arch}_virt"
-		;;
-	aarch64)
-		local nemu_bin="qemu-system-${arch}"
+		local nemu_bin="nemu-system-${arch}"
 		;;
 	*)
 		die "Unsupported architecture: $arch"
 		;;
 	esac
 
-	curl -LO "${nemu_repo}/releases/download/${nemu_version}/${nemu_bin}"
-	sudo install -o root -g root -m 0755 "${nemu_bin}" "/usr/local/bin"
-	rm -rf "${nemu_bin}"
+	mkdir -p "${GOPATH}/src"
```

Comment:
Is this actually needed or does `go get` create it if needed?

Reply:
Don't think that is needed. I added that for consistency. Dropping it should be fine.